spec/controllers/searches_controller_spec.rb

```diff
@@ -2,22 +2,16 @@
 
 RSpec.describe SearchesController, type: :controller do
   let!(:profile) { create(:profile, first_name: 'John') }
+  let!(:profile_subject) { create(:subject) }
+  let!(:with_subject) { profile.subjects << profile_subject }
```

Comment:
prefer to call creation here instead of `<<` subject to the profile, add factory trait `with_suject` and then
`let!(:profile) { create :profile, :with_subject }`